tidy3d/components/simulation.py

```diff
@@ -1401,6 +1403,12 @@ def subsection(
         include_pml_cells : bool = False
             Keep PML cells in simulation boundaries. Note that retained PML cells will be converted
             to regular cells, and the simulation domain boundary will be moved accordingly.
+        skip_geometry_validation: bool = False
```

Comment:
maybe `validate_geometries : bool = True` would be more straightforward in the code and as an API?